src/app/complex/complex-results/complex-filter/complex-filter.component.ts

```diff
@@ -133,4 +134,30 @@ export class ComplexFilterComponent implements OnInit {
   set interactorTypeFilter(value: string[]) {
     this._interactorTypeFilter = value;
   }
+
+  public interactorTypeIcon(facet): string {
```

Comment:
As we did with the `complex-navigator-utils.ts` file, we should have a file `complex-portal-utils.ts`, probably located at `src/app/complex/` where we can put the `interactorTypeIcon` function. It is used here and then in the complex navigator, so let's extract it to a common place.